packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -160,42 +160,12 @@ describe('Validation at REST level', () => {
 
     before(() =>
       givenAnAppAndAClient(ProductController, {
-        nullable: false,
         compiledSchemaCache: new WeakMap(),
         ajvKeywords: ['range'],
       }),
     );
     after(() => app.stop());
 
-    it('rejects requests containing `null` with {nullable: false}', async () => {
```

Comment:
So how can LB4 users request our REST validation layer to reject requests containing `null` value? Is that feature no longer supported? Are we going to set properties of `string` type to `null` values when a (malicious?) client sends payload with a `null` value?

I think this could be surprising to our users and potentially has security implications, therefore such change should be clearly described in our release notes.